api/handlers/dataexplorerhandler.py

```diff
@@ -308,6 +354,7 @@ def aggregate_field_values(self):
             }
         }
         if not filters:
+            # NOTE unreachable
```

Comment:
The code will need to get updated to support device behavior for completeness. That's where an empty filter could come from (no `uid` for device auth), but it doesn't currently support it. I'll open a ticket about supporting device requests in search.